cmd/bosun/sched/notify.go

```diff
@@ -59,6 +59,12 @@ func (s *Schedule) CheckNotifications(rh *RunHistory) time.Duration {
 			if st == nil {
 				continue
 			}
+			//if alert is currently unevaluated because of a dependency,
```

Comment:
Space after //. Comment is a sentence and should start with caps: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences